spec/bundle/dumper_spec.rb

```diff
@@ -16,15 +16,16 @@
     Bundle::WhalebrewDumper.reset!
     Bundle::BrewServices.reset!
 
-    chrome = instance_double("Cask::Cask", full_name: "google-chrome")
-    java = instance_double("Cask::Cask", full_name: "java")
+    chrome = instance_double("Cask::Cask", full_name: "google-chrome", to_s: "google-chrome")
+    java = instance_double("Cask::Cask", full_name: "java", to_s: "java")
+    iterm2beta = instance_double("Cask::Cask", full_name: "homebrew/cask-versions/iterm2-beta", to_s: "iterm2-beta")
```

Comment:
I'm not sure if this is the correct assertion as the issue we're having is with things like `cask "some/external/cask"` listed in the Brewfile ([example](https://github.com/jacobbednarz/j/blob/main/config/homebrew/Brewfile#L197)). I'm not entirely positive this solves the issue.

Reply:
In this case shouldn't I be checking a cask whose full name does include the tap and asserting that the output Brewfile doesn't contain one?

Reply:
Looks right to me 👍🏻

Reply:
Thanks for confirming!